less/angular/pages/home.less

```diff
@@ -307,4 +319,204 @@
       }
     }
   }
+  .variant-6 .home-panel {
+    padding: 0;
+    .container {
+      width: 100%;
+      padding: 0;
+    }
+    text-align: center;
+    .onboarding-section {
+      padding: 0;
+      margin: 0;
+    }
+    div {
+      display: inline-block;
+      width: 100%;
+      @media screen and (min-width: @screen-xs-min) {
+        width: auto;
+      }
+    }
+    .onboarding-step-2 {
+      padding: 0 2rem;
+      margin-bottom: 2rem;
+      @media screen and (min-width: @screen-xs-min) {
+        padding: 0px 5rem;
```

Comment:
This happens when I copy it from the browsers debugger, not sure why firefox adds the px to it, super annoying.